### DISCUSSION TITLE
Naming of packages

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#401] [Core Contributions] [answered]
The naming convention for most packages: autora-contribution_type-contribution_name. 
For example:
- autora-theorist-darts
- autora-experimentalist-inequality-sampler
But this one:
[autora-novelty-sampler](https://github.com/AutoResearch/autora-novelty-sampler) has a different naming convention (this also affects the pip release). 

*Option 1:*
I suggest renaming the autora-novelty-sampler to autora-experimentalist-novelty-sampler and keeping the above naming convention. The reason: 

When installing this package directly via pip, the user would have to write:
```shell
pip install autora-novelty-sampler
```
but when installing as subpackage from autora the command would be:
```shell
pip install -U autora["experimentalist-novelty-sampler"]

For the other packages, the conversion between these two follows a simple rule:
```shell
pip install autora-theorist-darts
```
=>
```shell
pip install -U autora["theorist-darts"]
```
*Option 2:*
We could also use only the last "sub-type". In this case, "sampler" instead of the full path and change the convention in the mother package as well, so that the installation as subpackage from autora looks like this:
```shell
pip install autora["novelty-sampler"]
```

*Option 3:*
No types at all, and just the contribution name. For example,
```shell
pip install autora["darts"]
```

* There certainly are other options.*

Suggestions or thoughts? Although the names are a bit long, I am in favor of *option 1*. In *option 1* the `pip install command`, the `folder structure`, the `python import statements` and the `github-repository name` align.

**Comment by benwandrew** [comment]
i also support option 1. i would rather pay the price of (a little) additional length for more clarity and better overall alignment

**Comment by younesStrittmatter** [comment, marked as answer]
The naming convention follows the folder and import structure: autora-experimentalist-sampler-<contribution-name>
